### DISCUSSION TITLE
Kotatogram Update

### DISCUSSION THREAD

**Discussion by mibnrashid** [kotatogram/kotatogram-desktop#410] [EN 2. Q&A]
Hey there, I like the work you've done on this app, but just would like to know are you thinking of updating anytime soon? Saw what you said already but like the latest version is 9 months old 🙂.
Thanks

**Comment by RememberTheAir** [comment]
![image](https://user-images.githubusercontent.com/16143208/209115567-43d9f41c-17a4-4973-bf87-10a44636e3a8.png)

**Reply by ilya-fedin** [reply]
![изображение](https://user-images.githubusercontent.com/17829319/209115958-967837d7-6386-4a26-bf12-79545223dde2.png)

**Comment by mibnrashid** [comment]
No😭???

**Reply by ilya-fedin** [reply]
Have you prayed hard enough? Have you tried to ask Santa?

**Reply by mibnrashid** [reply]
:( I'll take that as a no, so sad.

**Reply by ilya-fedin** [reply]
The update will be no earlier than @EricKotato gets free time. He has a full time job, that's why updates are rare. So they are possible only when there are long holidays (given the amount of work needed to rebase the client to latest tdesktop sources).

**Reply by mibnrashid** [reply]
Oh ok, I understand. Do you know any up-to-date similar builds for telegram? It's kind of annoying when I cant read some messages because it's not updated.
